build.gradle

```diff
@@ -243,32 +224,32 @@ tasks.distTar.enabled = false
 tasks.startScripts.enabled = false
 
 task createDistZip(type: Zip, dependsOn: createDist) {
-    archiveName = "IGV_${version}.zip"
+    archiveFileName = "IGV_${archiveVersion}.zip"
```

Comment:
The name changes are all to satisfy gradle deprecation complaints.

Reply:
OK, looks good